packages/amplify-graphql-model-transformer/src/resources/dynamo-model-resource-generator.ts

```diff
@@ -171,6 +171,10 @@ export class DynamoModelResourceGenerator extends ModelResourceGenerator {
     cfnTable.sseSpecification = {
       sseEnabled: cdk.Fn.conditionIf(useSSE.logicalId, true, false),
     };
+    cfnTable.timeToLiveSpecification = {
```

Comment:
For this specific line

Reply:
For customers using DataStore, they won't see any changes, but for customers not using DataStore or those who disable it, there will be this additional block with `enabled: false`.